tapdb/universe_forest.go

```diff
@@ -125,3 +130,69 @@ func (b *BaseUniverseForest) RootNodes(
 
 	return uniRoots, nil
 }
+
+// RegisterIssuance inserts a new minting leaf within the universe tree, stored
+// at the base key.
+func (b *BaseUniverseForest) RegisterIssuance(ctx context.Context,
```

Comment:
Can we also add fetching here? So then we'll be able to add that second level of proof and also verification.